src/Servers/HttpSys/src/HttpSysListener.cs

```diff
@@ -32,6 +33,8 @@ internal class HttpSysListener : IDisposable
         // 0.5 seconds per request.  Respond with a 400 Bad Request.
         private const int UnknownHeaderLimit = 1000;
 
+        internal MemoryPool<byte> MemoryPool { get; } = SlabMemoryPoolFactory.Create();
```

Comment:
Should we make this configurable to a user? I'd argue we should in case someone want's to, for example, not use shared memory for security concerns.

Reply:
Not yet, we should come up with a consistent way of doing this across the stack.

Reply:
It's not public in Kestrel either.